arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -2971,30 +6426,304 @@
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "NetInterfaceStatus",
+            "modelAsString": false
+          }
+        },
+        "isDefault": {
+          "description": "Value indicating whether this instance is default.",
+          "type": "boolean"
+        },
+        "iscsiAndCloudStatus": {
+          "description": "Value indicating cloud and ISCSI status of network adapter.",
+          "enum": [
+            "Disabled",
+            "IscsiEnabled",
+            "CloudEnabled",
+            "IscsiAndCloudEnabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ISCSIAndCloudStatus",
+            "modelAsString": false
+          }
+        },
+        "speed": {
+          "format": "int64",
+          "description": "The speed of the network adapter.",
+          "type": "integer"
+        },
+        "mode": {
+          "description": "The mode of network adapter, either IPv4, IPv6 or both.",
+          "enum": [
+            "Invalid",
+            "IPV4",
+            "IPV6",
+            "BOTH"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "NetworkMode",
+            "modelAsString": false
+          }
+        },
+        "nicIpv4Settings": {
+          "$ref": "#/definitions/NicIPv4",
+          "description": "The IPv4 configuration of the network adapter."
+        },
+        "nicIpv6Settings": {
+          "$ref": "#/definitions/NicIPv6",
+          "description": "The IPv6 configuration of the network adapter."
+        }
+      }
+    },
     "NetworkInterfaceData0Settings": {
-      "description": "Network interface Data 0 IP settings",
+      "description": "The 'Data 0' network interface card settings.",
       "type": "object",
       "properties": {
         "controllerZeroIp": {
-          "description": "Controller 0's IPv4 address",
+          "description": "The controller 0's IPv4 address.",
+          "type": "string"
+        },
+        "controllerOneIp": {
+          "description": "The controller 1's IPv4 address.",
+          "type": "string"
+        }
+      }
+    },
+    "NetworkSettings": {
+      "description": "Represents the network settings of a device.",
+      "required": [
+        "properties"
+      ],
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseModel"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NetworkSettingsProperties",
+          "description": "The properties of network settings of a device.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "NetworkSettingsPatch": {
+      "description": "Represents the patch request for the network settings of a device.",
+      "required": [
+        "properties"
```

Comment:
PatchBodyParametersSchemaValidation -> This Error, we had a discussion regarding it for DevicePatch Model(for Device_Update operation) in the First PR https://github.com/Azure/azure-rest-api-specs/pull/1098 . Similar to that, here too, we have just one property in the patch body parameter, "NetworkSettingsPatch", namely "properties" and we want it to be always passed. If we had more than one field in the NetworkSettingsPatch, I hope this check holds true.